src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,34 @@
 package core.basesyntax;
 
 public class MainApp {
+    public MainApp(String[] machine) {
 
+        Machine truck = new Truck() {
+            @Override
+
+            public void doWork() {
+                super.doWork();
+            }
+        };
+        truck.doWork();
+        truck.stopWork();
+        Machine excavator = new Excavator() {
+            @Override
+
+            public void doWork() {
+                super.doWork();
+            }
```

Comment:
Similarly, as with `Truck`, there's no need to override the `doWork` method for `Excavator` if you're not changing its behavior. You can instantiate `Excavator` directly.